src/lib/extended/module.ts

```diff
@@ -8,17 +8,17 @@
 import {NgModule} from '@angular/core';
 import {CoreModule} from '@angular/flex-layout/core';
 
-import {ImgSrcDirective} from './img-src/img-src';
-import {ClassDirective} from './class/class';
-import {ShowHideDirective} from './show-hide/show-hide';
-import {StyleDirective} from './style/style';
+import {DefaultImgSrcDirective} from './img-src/img-src';
+import {DefaultClassDirective} from './class/class';
+import {DefaultShowHideDirective} from './show-hide/show-hide';
+import {DefaultStyleDirective} from './style/style';
 
 
 const ALL_DIRECTIVES = [
-  ShowHideDirective,
-  ClassDirective,
-  StyleDirective,
-  ImgSrcDirective
+  DefaultShowHideDirective,
```

Comment:
If you are not exporting the Deprecated API, then they cannot be used. Is this your intent?

Reply:
Because they're not Angular `Directive`s, they don't need to be listed in the exports for the `NgModule`. They are still exported by the library and are still intended for use (they're not deprecated).